tools/wasme/cli/test/e2e/build/build_test.go

```diff
@@ -47,4 +57,69 @@ var _ = Describe("Build", func() {
 		err = test.WasmeCliSplit("push " + imageName)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	ExpectRustExampleToWorkInIstioVersion := func(version string) {
+		err := test.WasmeCliSplit("init test-filter --platform istio --platform-version " + version + ".x --language rust")
+		Expect(err).NotTo(HaveOccurred())
+		imagename := "testimage"
+		envoyimage := "docker.io/istio/proxyv2:" + version + ".0"
+		err = test.RunMake("builder-image")
+		Expect(err).NotTo(HaveOccurred())
+		err = test.WasmeCli(
+			"build",
+			"rust",
+			// need to run with --tmp-dir=. in CI due to docker mount concerns
+			"--tmp-dir=.",
+			"-t="+imagename,
+			"test-filter",
+		)
+		Expect(err).NotTo(HaveOccurred())
+
+		go func() {
+			defer GinkgoRecover()
+			err := test.WasmeCli(
+				"deploy",
+				"envoy",
+				imagename,
+				"--envoy-image="+envoyimage,
+				"--envoy-run-args=-l trace",
+				"--id=myfilter",
+			)
+			Expect(err).NotTo(HaveOccurred())
+		}()
+
+		testRequest := func() (string, error) {
+			b := &bytes.Buffer{}
+			w := io.MultiWriter(b, GinkgoWriter)
+			err := util.ExecCmd(
+				w,
+				w,
+				nil,
+				"curl",
+				"-v",
+				"http://localhost:8080/")
+			out := b.String()
+			return out, errors.Wrapf(err, out)
+		}
+
+		// expect header in response
+		// note that header key is capital case as this goes through Kube api
```

Comment:
Looks like this comment is no longer needed/true since we're running directly against docker?
```suggestion
```